components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -22,91 +22,108 @@ To get the best possible IAM v2 experience, Chef Automate should be running the
 Perform the upgrade to IAM v2 with `chef-automate iam upgrade-to-v2`; the response should look something like:
 
 ```terminal
+Upgrading to IAM v2.1
 Migrating v1 policies...
-
-Updating deployment configuration
-
-Applying deployment configuration
-  Started automate-gateway
-
 Creating default teams Editors and Viewers...
 
-Success: Enabled IAM v2
+Migrating existing teams...
+
+Success: Enabled IAM v2.1
 ```
 
-Note: To upgrade without porting existing policies, use the `--skip-policy-migration` flag: `chef-automate iam upgrade-to-v2 --skip-policy-migration`.
+To upgrade without porting existing policies, use: `chef-automate iam upgrade-to-v2 --skip-policy-migration`.
 
 ## View Policies
 
-After you've logged in to Chef Automate, select the **Settings** tab in the top navigation bar, then select and locate the `Policies` section on the left hand panel.
+After you've signed in to Chef Automate, select the **Settings** tab in the top navigation bar, then select and locate the `Policies` section in the left navigation.
 
 In this section, you can view all of your v2 policies. If you have upgraded without using the `--skip-policy-migration` flag, you will also see v1 policies.
 
 This includes the following:
 
-* New default (Chef-managed) policies: Administrator, Ingest, Editors, and Viewers.
+* New default, *Chef-managed* policies: Administrator, Ingest, Editors, and Viewers.
```

Comment:
I think this reads better without the comma, given that _all_ our default policies in v2 are Chef-managed, right?